src/MBC_TransactionalEmail_Consumer.php

```diff
@@ -186,6 +189,9 @@ protected function setter($message) {
       );
     }
 
+    if (isset($message['email-template'])) {
```

Comment:
Needed in case it's `email-template` rather than `email_template`. 

Product of a legacy bug / non-standard var name in various other producer apps that send messages to this consumer.